config/test.exs

```diff
@@ -0,0 +1,14 @@
+use Mix.Config
```

Comment:
I wonder, since it is config for tests, is it necessary to use all those `System.get_env`? Should we use the same port as dev config? (Was looking at my auto generated config from my phoenix exercise and they use different ports so I was wondering if it can affect the test 🤔 )

Reply:
well, I just copied and pasted it, but I don't see any problem of use the same config, it really depends, I guess.
what do you suggest to change?

Reply:
Hmm... do you think set/"hard-code" the config like the recommend in the [Hexdocs](https://hexdocs.pm/ecto/testing-with-ecto.html) would make any difference? I saw them has some `pool: Ecto.Adapters.SQL.Sandbox` I wonder how that could affect our test?

Reply:
let's try it out